#! /bin/sh /usr/share/dpatch/dpatch-run
## makefile_noiaxy.dpatch by Tzafrir Cohen <tzafrir.cohen@xorcom.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't try to install iaxy.bin, as it is not there.

@DPATCH@
diff -urNad asterisk-1.3.99~beta2.dfsg~/Makefile asterisk-1.3.99~beta2.dfsg/Makefile
--- asterisk-1.3.99~beta2.dfsg~/Makefile	2006-09-20 00:48:57.000000000 +0300
+++ asterisk-1.3.99~beta2.dfsg/Makefile	2006-10-08 02:07:08.000000000 +0200
@@ -401,7 +401,6 @@
 	$(INSTALL) -m 644 contrib/scripts/astgenkey.8 $(DESTDIR)$(ASTMANDIR)/man8
 	$(INSTALL) -m 644 contrib/scripts/autosupport.8 $(DESTDIR)$(ASTMANDIR)/man8
 	$(INSTALL) -m 644 contrib/scripts/safe_asterisk.8 $(DESTDIR)$(ASTMANDIR)/man8
-	$(INSTALL) -m 644 contrib/firmware/iax/iaxy.bin $(DESTDIR)$(ASTDATADIR)/firmware/iax/iaxy.bin; \
 
 $(SUBDIRS_INSTALL):
 	@$(MAKE) -C $(@:-install=) install
